backend/Squirrel.Core/Squirrel.Core.WebAPI/Extensions/ServiceCollectionExtensions.cs

```diff
@@ -25,6 +28,18 @@ public static void RegisterCustomServices(this IServiceCollection services)
         services.AddScoped<JwtIssuerOptions>();
         services.AddScoped<IJwtFactory, JwtFactory>();
         services.AddScoped<IAuthService, AuthService>();
+        services.Configure<MongoDatabaseConnectionSettings>(
+                    configuration.GetSection("MongoDatabase"));
```

Comment:
`services.Configure<MongoDatabaseConnectionSettings>(
                    configuration.GetSection("MongoDatabase"));` move it to `AddMongoDbService`